src/hls-adapter.js

```diff
@@ -274,6 +274,7 @@ export default class HlsAdapter extends BaseMediaSourceAdapter {
       this._config.hlsConfig['pLoader'] = pLoader;
     }
     this._maybeSetFilters();
+    this._config.hlsConfig.timelineController = CustomTimelineController;
```

Comment:
move it to `createAdapter` and only when `config.text.enableCEA708Captions` is `true`